### ISSUE TITLE
Feedback for "Set Up an HTTP Destination"

### ISSUE THREAD

**Issue by peterpersiel** [SAP-docs/btp-cloud-platform#44]
https://help.sap.com/products/BTP/65de2977205c403bbc107264b8eccf4b/3884bc38209843ac900d92adb9c2a863.html?locale=en-US&version=Cloud

> Instead of creating a destination from your own Destination service instance, you can configure destinations directly on subaccount level, in the subaccount in which the ABAP instance resides (recommended).

We should change this to 

> Instead of creating a destination from your own Destination service instance, you can configure destinations directly on subaccount level, in the subaccount in which the ABAP instance resides or for a SaaS solution in the consumer subaccount (recommended).

**Comment by Frank1Mueller** [comment]
Forwarded to Felix Draeger.

**Comment by felix95x** [comment]
The site has gone through some changes. You can find the updated version here:
https://help.sap.com/products/BTP/65de2977205c403bbc107264b8eccf4b/43a5ebed034547ad8aa85250f85098a2.html?state=DRAFT&version=Dev

It now says: 

> Optional: i_service_instance_name: Typically, you use the destinations of the subaccount in which the ABAP instance resides. However, you can add more destinations using your own destination service instance and communication scenario SAP_COM_0276, for example, to achieve separation of concerns (see also [Create a Destination](https://help.sap.com/products/BTP/65de2977205c403bbc107264b8eccf4b/3fa7934f5a714bf88d8490958211382f.html?state=DRAFT&version=Dev)). In this case, specify the value of the service instance name property of the communication arrangement for SAP_COM_0276.

Let me know if I can do any correction.

**Comment by peterpersiel** [comment]
Yes, please change the sentence "Typically, you use the destinations of the subaccount in which the ABAP instance resides" to "Typically, you use the destinations of the subaccount in which the ABAP instance resides or, in case of a SaaS solution based on ABAP Environment, the destinations of the consumer subaccount." This is necessary because in case of a SaaS solution the destinations have to be in a different subaccount from where the ABAP instance resides in.
